src/main/java/com/codegym/shopyy/controller/ProductDetailController.java

```diff
@@ -0,0 +1,66 @@
+package com.codegym.shopyy.controller;
+
+import com.codegym.shopyy.model.ProductDetail;
+import com.codegym.shopyy.service.IProductDetailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@Controller
+@CrossOrigin("*")
+@RequestMapping("/productDetail")
```

Comment:
rename ProductDetailController to ProductController
change /productDetail to /api/products/detail